src/app/(frontend)/ticket/[ticketCode]/page.client.tsx

```
@@ -50,110 +51,129 @@ export function TicketDetails({ ticket, isCheckedIn }: { ticket: Ticket; isCheck
-        </div>
 
-        {formattedDateTime && (
-          <p className="mt-2 flex items-start text-sm text-gray-600">
-            <Calendar className="w-4 h-4 mr-2 mt-0.5 flex-shrink-0" />{' '}
-            <span>{formattedDateTime}</span>
-          </p>
-        )}
+          {event?.title && (
+            <h2
+              className="text-2xl font-bold"
+              dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(event.title) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)